morse_code.rb

```diff
@@ -0,0 +1,41 @@
+MORSE_KEY = {
+  '.-' => 'A',
+  '-...' => 'B',
+  '-.-.' => 'C',
+  '-..' => 'D',
+  '.' => 'E',
+  '..-.' => 'F',
+  '--.' => 'G',
+  '....' => 'H',
+  '..' => 'I',
+  '.---' => 'J',
+  '-.-' => 'K',
+  '.-..' => 'L',
+  '--' => 'M',
+  '-.' => 'N',
+  '---' => 'O',
+  '.--.' => 'P',
+  '--.-' => 'Q',
+  '.-.' => 'R',
+  '...' => 'S',
+  '-' => 'T',
+  '..-' => 'U',
+  '...-' => 'V',
+  '.--' => 'W',
+  '-..-' => 'X',
+  '-.--' => 'Y',
+  '--..' => 'Z'
+}.freeze
+
+def decode_char(morse_code)
+  MORSE_KEY[morse_code]
+end
+
+def decode_word(morse_code)
+  word = ''
+  morse_code_splited = morse_code.split(/\s/i)
+  morse_code_splited.each do |letter|
+    word += decode_char(letter)
+  end
+  word
+end
```

Comment:
- [ ] Good work on creating these two methods. however, there is another requirement to create the third method `decode` which will decode the entire message. kindly consider creating this third method.
- Create a method to decode the entire message in Morse code, takes a string parameter, and return the string 
representation. Every word will be separated by 3 spaces (e.g.

  decode("-- -.--   -. .- -- .")
returns "MY NAME").